content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/headlibs.html

```diff
@@ -13,7 +13,7 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
-<template data-sly-template.headlibs="${@ designPath, staticDesignPath, clientlibCategories, hasCloudconfigSupport}"
```

Comment:
I think this has the potential of breaking other people's existing customizations, since we are changing the order of parameters. It's probably safer to add the new option at the end.

Reply:
Just ignore, the expression options must be passed by name anyway.